content/en/docs/2024.3/Reference/data-types/ssh/authentication/SshCredentials.md

```diff
@@ -1,12 +1,67 @@
 ---
 title: "SshCredentials"
 linkTitle: "SshCredentials"
-description: "Used to represent details required to authenticate with a server."
+description: "Any data type used to represent details required for authentication when working with SSH."
 weight: 1
 ---
 
 # {{% param title %}}
 
 <p class="namespace">(Cortex.DataTypes.Ssh.Authentication.SshCredentials)</p>
 
-{{< workinprogress >}}
+{{% alert type="information" title="Information" %}} Improvements to this page are planned for the future. {{% /alert %}}
+
+## Summary
+
+Any data type used to represent details required for authentication when working with SSH.
```

Comment:
Again, I thought this was a single data type not any data types

Reply:
this is an abstract class, so anywhere where this is mentioned it will actually be referring to the classes that implement it.